public/wp-content/themes/glific/functions.php

```diff
@@ -38,6 +38,32 @@ function glific_theme_setup(){
 	add_action('init','glific_theme_setup');
 }
 
+function get_youtube_video_id($youtube_url) {
+	parse_str( parse_url($youtube_url, PHP_URL_QUERY), $url);
+	$youtube_video_id = $url['v'];
+	return $youtube_video_id;
+}
+
+function get_youtube_video_title($youtube_video_id) {
+	$apikey = WP_YOUTUBE_API_KEY;
+	if (!empty($apikey)) {
+		$json_data = file_get_contents('https://www.googleapis.com/youtube/v3/videos?id='.$youtube_video_id.'&key='.$apikey.'&part=snippet');
```

Comment:
Can you put this URl  in Config?
`https://www.googleapis.com/youtube/v3/videos`